backend/models/entry.js

```diff
@@ -0,0 +1,79 @@
+import Image from './image'
+import DataTypes from 'sequelize'
+import sequelize from '../config/sequelize'
+
+const IMAGE = 3
```

Comment:
Is it worth it to make these enums and move them out to another file?

Reply:
Will do